frontend/src/components/users/Users.tsx

```diff
@@ -6,44 +6,46 @@ import Search from "components/search/Search";
 export type IUsers = IUser[];
 
 const Users = () => {
-	const { handleChange, filteredUsers, users } = useUsers();
+    const { handleChange, filteredUsers, users } = useUsers();
 
-	const UsersList = () => {
-		return (
-			<>
-				{users.length === 0 ? (
-					<span style={{ color: "white" }}>Loading...</span>
-				) : (
-					<Users />
-				)}
-			</>
-		);
-	};
+    const Users = () => (
+        <>
+            {filteredUsers.map(({ id, username }: IUser) => (
+                <User
+                    key={id}
+                    id={id}
+                    username={username}
+                />
+            ))}
+        </>
+    );
 
-	const Users = () => (
-		<>
-			{filteredUsers.map(({ id, username }: IUser) => (
-				<User
-					key={id}
-					id={id}
-					username={username}
-				/>
-			))}
-		</>
-	);
+    const UsersList = () => {
+        return (
+            <>
+                {users.length === 0 ? (
+                    <div style={{ color: "white", paddingTop: "1rem" }}>
```

Comment:
Imo it should be p tag, not div

Reply:
i think about it, but that's for now, like that, i change it for 'p' tag, but now it's unnecessary to create it as styled component, cause we want to create loading for all necessary places on our side #41